ganga/GangaCore/Utility/feedback_report.py

```diff
@@ -23,15 +25,12 @@ def deny_modification(name, x):
         pass
 _initconfigFeed()
 
-def send_request(url,data,files):
-    response = requests.post(url, data=data, files=files)
-    return response
-def report(job=None):
+def report(job=None, filetype=None):
```

Comment:
I think we should pick a default value for the `filetype`. `GoogleFile` is probably a good default as it allows for the easy sharing.